File: Wprowadzenie do funkcji/IV. Wprowadzenie do funkcji/zadanie2.cpp
#include <iostream>
using namespace std;

bool zawiera_wyraz(string tekst, string wyraz)
{
    for(int i = 0; i < tekst.length() - wyraz.length() + 1; i++)
    {
        bool ok = true;
        for(int j = 0; j < wyraz.length(); j++)
        {
            if(tekst[i + j] != wyraz[j])
            {
                ok = false;
                break;
            }
        }

        if(ok)
        {
            return true;
        }
    }

    return false;
}

int main()
{
    return 0;
}

File: Wprowadzenie do funkcji/IV. Wprowadzenie do funkcji/zadanie2.py
def zawiera_wyraz(tekst, wyraz):
    for i in range(0, len(tekst) - len(wyraz) + 1):
        ok = True
        for j in range(0, len(wyraz)):
            if tekst[i+j] != wyraz[j]:
                ok = False
                break
        if ok:
            return True
    return False